tools/rna_tools/dot2ct/datatypes_conf.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<datatypes>
+	<datatype_files>
+		<datatype_file name="RNAStructure.py"/>
+	</datatype_files>
```

Comment:
Awesome, we will have today a datatype discussion: https://github.com/galaxy-iuc/standards/issues/13
So we will know soon if we can move this straight into Galaxy main.